### DISCUSSION TITLE
mod_odbc_query

### DISCUSSION THREAD

**Discussion by Globalebill** [custompbx/custompbx#17] [Q&A]
how might i add this module to /cweb/configuration/modules 

module is built from src in freeswitch

**Comment by zusrut** [comment]
Hello @Globalebill. It need to be added by coding backend and frontend. I will check this module config to add it later. Until module not added to the custompbx it still can be managed by static xml file in freeswitch configs directory.

**Reply by Globalebill** [reply]
hi @zusrut i see on the dialplan page there is name and data fields, the odbc query module and other sometimes have a 3rd var in the xml such as "inline" as shown below:
action application="odbc_query" data="get-service" inline="true"
is this possible to add ?

**Reply by zusrut** [reply]
Hi @Globalebill  - when you put in action field value = odbc_query inline field will apear 🪄
it done for all actions which can have inline attribute

PS. the field was too narrow because of angular material update - just fixed it
